src/Components/Home.js

```diff
@@ -22,6 +24,18 @@ function Home() {
   return (
     <main>
       {showProgress && <ProgressBar mode="indeterminate" />}
+      {skeleton
+        ? <div>
+        <div style={{ display: 'none' }}>
+      {setTimeout(() => {
+        setskeleton(false)
+      }, 3000)}
+      </div>
+      <div className="p-d-flex p-flex-wrap">
+          {list.map((user, index) => { return <Skeleton shape="circle" size="65px" className="p-mr-2 p-m-2" key={index} /> })}
+      </div>
+      </div>
+        : <>
```

Comment:
the large ternary is difficult to read, it might be easier to separate into functions